libs/@local/harpc/client/typescript/src/net/Connection.ts

```diff
@@ -227,7 +227,13 @@ export const makeUnchecked = (
       ),
       Stream.mapConcat((list) => list),
       // cast needed as uint8arraylist doesn't support Uint8Array<ArrayBuffer> yet
```

Comment:
I guess this would also be the old behavior then?
```suggestion
```

Reply:
This is still the case, the generic for `Uint8Array` is relatively new and uint8arraylist doesn't make use of it yet, therefore we need to still cast. The current type stipulates that also a `SharedArrayBuffer` could be used, which isn't correct.